cocotb/build_libs.py

```diff
@@ -0,0 +1,476 @@
+#!/usr/bin/env python
```

Comment:
```suggestion
```
This looks like it is a module to import, not a script - so it should not have a shebang